lib/model.js

```diff
@@ -2190,10 +2190,12 @@ class Model {
 
       // Attach createdAt
       if (createdAtAttr && !updateValues[createdAtAttr]) {
-        insertValues[createdAtAttr] = this._getDefaultTimestamp(createdAtAttr) || now;
+        const attribute = this.rawAttributes[createdAtAttr].field || createdAtAttr;
+        insertValues[attribute] = this._getDefaultTimestamp(attribute) || now;
```

Comment:
If attribute has a field set, this will accidentally pass that field to `_getDefaultTimestamp` where instead we want to pass attribute in all cases